classes.py

```diff
@@ -0,0 +1,7 @@
+class Process:
+        
+    def __init__(self, path, process):
```

Comment:
The process can probably be done by the program itself, if possible we should do all `subprocess` calls in this file, delegate the responsibility. This way it'll make `startUp.py` easier to test as well since we can make a mock `process`.